build/common/installer/scripts/tomlparser-prom-agent-config.rb

```diff
@@ -14,7 +14,7 @@
 @promFbitMemBufLimit = 200
 
 @waittime_port_25226 = 45
-@waittime_port_25228 = 60
+@waittime_port_25228 = 120
```

Comment:
want to double check, this change doesnt impact the initial liveness probes, right?

Reply:
yes based on the understanding all the necessary livenessprobe executables have been started before this so shouldnt't affect